<launch>
  <arg name="debug" default="false"/>
  <!--<arg name="debug_valgrind" default="false"/>-->

  <arg name="prefix" value="" unless="$(arg debug)"/>
  <arg name="prefix" value="gdb -ex run --args" if="$(arg debug)"/>

  <node pkg="mpl_test_node"
    type="test_rss"
    name="test_primitive"
    launch-prefix="$(arg prefix)"
    output="screen">
    <param name="file" value="$(find mpl_test_node)/maps/simple/simple.bag"/>
    <param name="topic" value="/voxel_map"/>
    <!-- Set start and goal -->
    <param name="start_x" value="11.0"/>
    <param name="start_y" value="3"/>
    <param name="start_z" value="0.0"/>
    <param name="start_vx" value="2.0"/>
    <param name="start_vy" value="0.0"/>
    <param name="start_vz" value="0.0"/>
    <param name="goal_x" value="6.4"/>
    <param name="goal_y" value="16.6"/>
    <param name="goal_z" value="0.0"/>
    <!-- Set dynamic constraints -->
    <param name="v_max" value="4.0"/>
    <param name="a_max" value="1.0"/>
    <param name="dt" value="1.0"/>
    <param name="ndt" value="2"/>
  </node>

</launch>
